x/configurl/doc.go

```diff
@@ -103,12 +103,35 @@ For more details, refer to [github.com/Jigsaw-Code/outline-sdk/transport/tlsfrag
 
 	tlsfrag:[LENGTH]
 
+Disorder transport (streams only, package [github.com/Jigsaw-Code/outline-sdk/x/disorder])
```

Comment:
This is duplicated.

Let's keep it once, after the split.

I'd call it "Packet reordering" instead of "Disorder transport", to make a parallel to the splitting.